tutorials/packaging/continuous-snap-delivery-from-circle-ci/continuous-snap-delivery-from-circle-ci.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: continuous-snap-delivery-from-circle-ci
-summary: Learn how to use Circle CI to deliver a new snap version to your early adopters every time you make a change in your master branch.
+id: continuous-snap-delivery-from-circleci
```

Comment:
@felicianotech would you be OK with not changing the ID?

Reply:
Yup. Do you need me to do that or will you? I believe edits are on.